lib/appenders/file.js

```diff
@@ -53,6 +53,15 @@ function fileAppender(file, layout, logSize, numBackups, options, timezoneOffset
   let writer = openTheStream(file, logSize, numBackups, options);
 
   const app = function (loggingEvent) {
+    if (options.removeColor === true) {
+      // eslint-disable-next-line no-control-regex
+      const regex = new RegExp("\x1b[[0-9;]*m", "g");
```

Comment:
Use the regexp literal (e.g. `regex = /\x1b[..../g`) instead of `new RegExp` constructor when the pattern won't change. The literal gets evaluated once, whereas the constructor will create a new one every time.